front-end/src/components/Pages/Billing.js

```diff
@@ -3,8 +3,8 @@ import { connect } from "react-redux";
 import { Redirect } from "react-router-dom";
 import Sidebar from "../Organisms/SideNav.js";
 
-import "../../styles/sidebar.css";
-import "../../styles/billing.css";
+import { Container, FormItem, Form } from "../../styles/signin.js";
```

Comment:
Just a note: Fine for now, but maybe in the future we will start some global styles and re-use them across the app.